src/tokens-studio/spectrum2-colors/$themes.json

```diff
@@ -571,7 +570,8 @@
       "Palette.transparent-white.800": "ea5e834dbe202dc1dbbce86ba074ab50155812c1",
       "Palette.transparent-white.900": "cc73dfce811bf9bab293a95d9342fd6a9135c9ab",
       "Palette.transparent-white.1000": "0ae4fb367bd6d70103b8df1fd1fff8a237a94b30",
-      "Alias.🚫 drop-shadow": "0e6900e9ce8ddf2c72e312dcc202679dd76f2516"
+      "Alias.🚫 drop-shadow": "0e6900e9ce8ddf2c72e312dcc202679dd76f2516",
```

Comment:
@nabuhasan I noticed this one was also flagged as modified. it doesn't look like there are any actual changes though, maybe it's due to reordering? do you have an idea of why this is flagged here? 🤔

Reply:
It might be due to something I did. I initially placed the deprecation icon on "Component"
in the Component.color-loupe.drop-shadow token in TS. Then I updated it to: Component.color-loupe.🚫drop-shadow. 

It might be due to that slight switch up.

Reply:
@lynnhao @nabuhasan I think, its because the ""Component.color-loupe.drop-shadow" was moved to the end of the list, and the line with "Alias.🚫 drop-shadow" has a comma ',' at the end now. 
So you can ignore this change, because its just saying, there is a new item in the next line.